bgflow/bg.py

```diff
@@ -2,12 +2,12 @@
 
```

Comment:
the only change in this file is that the distinctions between tensors and tuples-of-tensors are now handled by the new `pack_tensor_in_tuple` function.